forge-gui/res/cardsfolder/upcoming/revive_the_shire.txt

```diff
@@ -0,0 +1,8 @@
+Name:Revive the Shire
+ManaCost:1 G
+Types:Sorcery
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Hand | ValidTgts$ Permanent.YouCtrl | SubAbility$ DBFood | SpellDescription$ Return target permanent card from your graveyard to your hand. Create a Food token. (It's an artifact with "{2}, {T}, Sacrifice this artifact: You gain 3 life.")
+SVar:DBFood:DB$ Token | TokenAmount$ 1 | TokenScript$ c_a_food_sac
+DeckHas:Ability$Token|LifeGain|Graveyard & Type$Artifact|Food
+DeckHints:Type$Halfling
```

Comment:
Hints synergy...?

Reply:
Yeah more a flavor hint because of food + art and name but will remove